packages/libs/client-utils/src/marmalade/get-auction-details.ts

```diff
@@ -0,0 +1,59 @@
+import type { IPactModules, PactReturnType } from '@kadena/client';
+import { Pact } from '@kadena/client';
+import { composePactCommand, execution, setMeta } from '@kadena/client/fp';
+import type { ChainId } from '@kadena/types';
+import { dirtyReadClient } from '../core/client-helpers';
+import type { IClientConfig } from '../core/utils/helpers';
+import type { IAuctionConfig } from './config';
+
+interface IGetAuctionDetailsInput {
+  auctionConfig: IAuctionConfig;
+  saleId: string;
+  chainId: ChainId;
+  guard: {
+    account: string;
+  };
+}
+
+const getAuctionDetailsCommand = ({
```

Comment:
`get-auction-details` need not be a `/send` tx, but will most of the time be called as a `/local` tx, so I don't think we need to input a `guard` in a getter function.

Reply:
Same comment would go to all the `get-*` functions and the functions not to be sent to the blockchain: 
`create-bid-id`, `create-token-id`, `escrow-account`, `get-account-details`, `get-bid`, `get-current-price`, `get-escrow-account`, `get-token-balance`, `get-token-info`, `get-uri`

Reply:
It's already using a `/local` call using `dirtyReadClient` but you're right, we don't need `guard` so I removed it.